components/monitoring/prometheus/README.md

```diff
@@ -1,34 +1,67 @@
-## Data Plane Clusters Prometheus Instances
-We use the Openshift-provided Prometheus deployments, platform and
-user-workload-monitoring (UWM), alongside a Prometheus instance deployed by the RHOBS
+## RHTAP Central Monitoring
+The RHTAP monitoring solution is based on three Prometheus instances deployed to each
+Production and Staging cluster. Each cluster writes a subset of the metrics it generates
+into Observatorium (RHOBS), marking each metric with a label containing its cluster of origin.
+
+Observatorium holds two different tenant for RHTAP, one for Production and one for
+Staging. The metrics collected for each of those tenants are made available to the
+AppSRE team via the AppSRE Grafana instance.
```

Comment:
Is the role of this only to make things available for AppSRE? If not, lets frame things differently.

Reply:
done